src/sentry/static/sentry/app/views/releases/detail/overview/chart/index.tsx

```diff
@@ -45,40 +46,54 @@ const ReleaseChartContainer = ({
 }: Props) => {
   const {projects, environments, datetime} = selection;
   const {start, end, period, utc} = datetime;
-  const eventView = getReleaseEventView(selection, version);
+
+  let chart: React.ReactNode = null;
+
+  if (hasDiscover && (yAxis === YAxis.EVENTS || yAxis === YAxis.FAILURE_RATE)) {
```

Comment:
Should this be gating on the account having 'performance-view' instead of discover?

Reply:
That's a good point, this was just a first pass to get it up on staging so we can play with the data but gating it on `performance-view` instead makes more sense.